server/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:17.3.0-alpine AS development
+
+RUN mkdir -p /usr/app/server
+WORKDIR /usr/app/server
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+FROM node:17.3.0-alpine as production
```

Comment:
better nit to use non LTS versions.
For NodeJS odd versions are LTS.
Like 14/16/18.
You can use 18 for now